ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/HolderAccess.java

```diff
@@ -0,0 +1,55 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "holder_access")
+@Data
+@NoArgsConstructor(force = true)
+public class HolderAccess extends BankAccountAccess {
+
+
+//    @Column(name = "holder_id")
+    @Column(name = "holder_id")
+    private String  holderId; // ID of the account holder
+
+    @Column(name = "user_id", nullable = false)
+    private String userId;
```

Comment:
what is the difference  between a ```userId and a holderId``` ??

when a user creates an account, the user is being assigned an ```accountId``` which is like a bank account id and an ```entityId```

these ```entityId``` could be the  specific id of the account holder, it  could be the id for maybe an Agency..

keep in mind that all class extend from the ```Entity BankAccountAccess```..

**See Here** :
When a user creates an account, they are assigned an accountId (the bank account ID) and an entityId.
The ```entityId``` could represent the ```holderId``` if the user is the account holder(```userId```), or it could represent an agency or organization if the account is being created for a business purpose.
All classes extending from the **Entity BankAccountAccess** would likely use these identifiers to manage access rights and permissions effectively.

Hope we are in phase...tell me what you think